### DISCUSSION TITLE
How to build .lotus to a separate volume?

### DISCUSSION THREAD

**Discussion by Nicholas-Zarate** [filecoin-project/lotus#8808] [Build and Sync] [answered]
Running on Ubuntu Server 22.04 LTS

**I have a 2 disks:**

(30GiB) Disk 1: /root and my /boot partition live 

(1 TiB) Disk 2: /lotus folder and chain snapshot

**Issue:**

With:
`sudo make install`

**I want to install /.lotus in Disk 2 along with my proof parameters so my chain doesn't fill up Disk 1**

**How do I do that? Is this a good idea? I don't want everything on my boot drive since it is only 30Gib. I want as much as possible on my Disk 2** 

I am also told .lotus installs into $HOME/.lotus so that may be a configuration I need to change maybe through altering the Makefile or adding a prefix. I tried adding a prefix to my `sudo make install` but that did not work.

I'm referencing a Tutorial discussion [HERE](https://github.com/filecoin-project/lotus/discussions/5989) where stuberman lays out his disk arrangement. I have pasted below the section I am referencing.

> Disk 1 (500 GB)
/boot partition (500MB should be more than enough)
/ (root) this is where your OS goes (10GB), perhaps also your /home for the lotus source code (10GB) and a directory for your proof parameters (158GB) - so a minimum of 200GB preferably fast disk for the proofs
/swap with only 128GB ram you will need a lot os fast swap (200GB perhaps on its own partition)

>Disk 2 (1 TB)
/.lotus your chain will grow quick, as I write this it is growing at 38 GB/day, never let if fill the disk, expect to shut down the daemon and miner to reimport a pruned chain around 47 GB)

Any tips or resources would be appreciated. I'm new to Lotus, would appreciate all the help I can get :)

**Comment by stuberman** [comment, marked as answer]
Simply set an ENV variable before you install lotus:

`export LOTUS_PATH=`

[Lotus docs](https://lotus.filecoin.io/storage-providers/setup/initialize/#checklist)
